doc/source/whatsnew/v0.24.0.txt

```diff
@@ -181,7 +181,7 @@ Reshaping
 Other
 ^^^^^
 
--
+- :meth: `~pandas.io.formats.style.Styler.background_gradient` now takes a ``text_color_threshold`` parameter to automatically lighten the text color based on the luminance of the background color. This improves readability with dark background colors without the need to limit the background colormap range. (:issue:`21258`, :issue:`21269`)
```

Comment:
Can just reference the first issue here, since that is what you are closing anyway (second was a duplicate)